### ISSUE TITLE
Regression in 2.18.0

### ISSUE THREAD

**Issue by roryabraham** [cloudflare/python-cloudflare#184]
We started [consistently seeing errors clearing our CloudFlare cache](https://github.com/Expensify/App/actions/workflows/platformDeploy.yml), and the timing coincides closely with the release of 2.18.0, which suggests there may have been some regression introduced.

The command we're doing is:

```sh
cli4 --delete hosts=["staging.new.expensify.com"] /zones/:9ee042e6cfc7fd45e74aa7d2f78d617b/purge_cache
```

**Comment by roryabraham** [comment]
I apologize for the low-quality bug report, but there's nothing being logged, just a command exiting with exit code 1:

![image](https://github.com/cloudflare/python-cloudflare/assets/47436092/5773629c-04a2-48b7-a1e3-34b510698ec8)

we've also verified that the credentials being used are correct and up-to-date.

**Comment by roryabraham** [comment]
Going to confirm this by pinning our CI to 2.17.0

**Comment by roryabraham** [comment]
Confirmed that there was some regression in 2.18.0 as pinning to 2.17.0 fixed the issue for us. We also enabled verbose logging in our CI so we'll be able to produce better bug reports.

**Comment by mahtin** [comment]
Yikes! (But I think I saw a `purge_cache` issue earlier this year independent of release cycle - I'll try to find it).

A simple `-v` flag on a manual run of your `cli4` command would really help debug this. (Both on this and the previous release). Thanks.

**Comment by mahtin** [comment]
@roryabraham - `2.18.0` was yanked a few days ago. Please try `2.18.2`.

**Comment by mahtin** [comment]
Bug found (after a long goose chase). It was in all `2.18.?` releases. I've fixed and pushed `2.19.0`. Please confirm it's fixed for you. In fact, I have yanked all the `2.18.?` releases as they all had the same bug in the `cli4` command (but the library was fine). Sorry about that.

**Comment by roryabraham** [comment]
Thanks @mahtin, we'll try out 2.19.0 and report back. We have also enabled verbose logging so we'll be able to provide better bug reports in the future if they occur.

**Comment by roryabraham** [comment]
Thanks @mahtin, confirmed this issue is fixed in `2.19.0`. Thanks again for the quick fix!
